ydb/core/mind/bscontroller/scheme.h

```diff
@@ -49,7 +49,7 @@ struct Schema : NIceDb::Schema {
     };
 
     struct Group : Table<4> {
-        struct ID : Column<1, NScheme::NTypeIds::Uint32> {}; // PK
+        struct ID : Column<1, NScheme::NTypeIds::Uint32> { using Type = TIdWrapper<ui32, TGroupIdTag>; static constexpr Type Default = TIdWrapper<ui32, TGroupIdTag>::Zero();}; // PK
```

Comment:
TIdWrapper<ui32, TGroupIdTag> should be mentioned only once as a global definition